### ISSUE TITLE
Adding commonly used keywords for the Makie recipe?

### ISSUE THREAD

**Issue by henry2004y** [SciML/SciMLBase.jl#702]
As explained in https://docs.makie.org/v0.21/explanations/specapi#Using-specs-in-convert_arguments, it's not decided yet how to forward keyword arguments from `plots(...; kw...)` to convert_arguments for the `SpecApi` in a more convenient and performant way in Makie. 

Currently the only supported keywords for the SciML Makie recipe are

https://github.com/SciML/SciMLBase.jl/blob/de519140ea8e1e87d531aee30afe257425c6992a/ext/SciMLBaseMakieExt.jl#L140-L143

I'm wondering if it is possible to some commonly used keyword arguments as a temporary workaround. For example, `color`, `linewidth`, `linestyle`, etc.

Related issue: https://github.com/MakieOrg/Makie.jl/issues/3623

**Comment by ChrisRackauckas** [comment]
Yes this is the main reason why Makie recipes are not prime time for us right now. I am not 100% sure if we have the tools with current Makie recipes to solve it, but if we do it would be nice to get some help to show us.
